From b9eaf1a744f05abb15ca0e47a0d1e96351a0f3e6 Mon Sep 17 00:00:00 2001
From: Li Jun <b47624@freescale.com>
Date: Mon, 12 Jan 2015 15:58:21 +0800
Subject: [PATCH] MLK-10085-3 ARM: imx6: remove imx6-usb-charger-detection for
 imx6sx-sabreauto

Disable imx6-usb-charger-detection by default.

Signed-off-by: Li Jun <jun.li@freescale.com>
---
 arch/arm/boot/dts/imx6sx-sabreauto.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6sx-sabreauto.dts b/arch/arm/boot/dts/imx6sx-sabreauto.dts
index 14c7adb..2991ef3 100644
--- a/arch/arm/boot/dts/imx6sx-sabreauto.dts
+++ b/arch/arm/boot/dts/imx6sx-sabreauto.dts
@@ -838,7 +838,6 @@
 	vbus-supply = <&reg_usb_otg1_vbus>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_usbotg1_1>;
-	imx6-usb-charger-detection;
 	status = "okay";
 };
 
-- 
1.8.0

